README.rst

```diff
@@ -88,7 +88,7 @@ You can run tests by executing::
 
     virtualenv env
     source env/bin/activate
-    pip install -r tests/requirements.txt
```

Comment:
please change to tests/requirements/base as in djangocms-googlemap

Reply:
I leave `tests/requirements.txt` because we don't need more requirements right now so the flat is better.